### ISSUE TITLE
[Bug]: Total count host when filtered

### ISSUE THREAD

**Issue by vvatelot** [vvatelot/ecoindex_monorepo#22]
### What happened?

The total count match the number of analyzis et not hosts...

### Project

Ecoindex API

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by PaulPHPE** [comment]
Hello @vvatelot ! 

I am not sure to understand this issue, is it related to this request : https://api.ecoindex.fr/docs#/Host/Get_host_list__version__hosts_get ?

**Comment by vvatelot** [comment]
Yes, for example if you make a request with filter `host=ecoindex` the total count still be the same as if there is no filter...
